models/aeif_cond_beta_multisynapse.cpp

```diff
@@ -466,13 +464,20 @@ aeif_cond_beta_multisynapse::calibrate()
 
   for ( size_t i = 0; i < P_.num_of_receptors_; ++i )
   {
-    double t_p = P_.taus_decay[ i ] * P_.taus_rise[ i ]
-      / ( P_.taus_decay[ i ] - P_.taus_rise[ i ] )
-      * std::log( P_.taus_decay[ i ] / P_.taus_rise[ i ] ); // peak time
-    V_.g0_ex_[ i ] = V_.g0_in_[ i ] // normalization factors for conductance
-      = ( 1. / P_.taus_rise[ i ] - 1. / P_.taus_decay[ i ] )
-      / ( std::exp( -t_p / P_.taus_decay[ i ] )
-          - std::exp( -t_p / P_.taus_rise[ i ] ) );
+    if ( P_.taus_decay[ i ] == P_.taus_rise[ i ] ) // alpha function limit
```

Comment:
Testing doubles for equality is not really safe. What you in the end need is that
```
P_.taus_decay[ i ] - P_.taus_rise[ i ] != 0
```
and
```
std::exp( -t_p / P_.taus_decay[ i ] ) - std::exp( -t_p / P_.taus_rise[ i ] ) != 0
```
These expressions may be zero even if the rise and decay time differ slightly. `doc/model_details/IAF_neurons_singularity.ipynb` discusses how we handle singularities due to `tau_m ~ tau_syn` in linear models; you may want to take a look there.